docs/BUILD.bazel

```diff
@@ -8,7 +8,11 @@ stardoc_with_diff_test(
     symbol_names = ["jest_test"],
 )
 
-update_docs(name = "update")
+update_docs(
+    name = "update",
+    # slight docs difference in Bazel 6
+    tags = ["skip-on-bazel6"],
```

Comment:
is there a way to do it with `target_compatible_with`

Reply:
Not that I know of